apps/vor/frontend/src/api/students/useGetStudents.ts

```diff
@@ -33,6 +33,6 @@ async function fetchStudents(): Promise<Student[]> {
   return result.json()
 }
 
-export const useGetStudents = (): QueryResult<Student[]> => {
-  return useQuery("students", fetchStudents)
+export const useGetStudents = (classId = ""): QueryResult<Student[]> => {
+  return useQuery(classId, fetchStudents)
```

Comment:
I think it would be better if the key is `["students", classId]` instead of just `classId` to avoid collisions with other keys and to be consistent with other api hooks.
```suggestion
  return useQuery(["students", classId], fetchStudents)
```